DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Helpers/RevitCategories.cs

```diff
@@ -0,0 +1,148 @@
+using System;
+using System.Collections.Generic;
+using Autodesk.Revit.DB;
+
+namespace Speckle.Converters.RevitShared.Helpers;
+
+/// <summary>
+/// Contains predefined categories of revit objects as well as the types that make up the elements
+/// in that category
+/// </summary>
+public static class RevitCategories
```

Comment:
Hmm I imagine this is pre-existing but the mass of statics feels a bit messy, we can perhaps look at it later. At the very least we ought to put some whitespace between each static